package.json

```diff
@@ -120,6 +120,23 @@
             "default": true,
             "description": "%configuration.SAS.session.outputHtml%"
           },
+          "SAS.results.html.style": {
+            "order": 1,
+            "type": "string",
+            "default": "(auto)",
+            "enum": [
+              "(auto)",
+              "(server default)",
+              "HighContrast",
+              "Ignite",
+              "Illuminate"
```

Comment:
I'm waiting to hear back from others on which of the ODS styles we should include here in the option UI. I'm thinking "Ignite", "Illuminate", and "HighContrast" at a minimum. I ran the SAS code "proc sql; select * from sashelp.vstyle; quit;" on latest Viya 4 (2023.09) using SAS Studio and there were 60 ODS styles were defined. I was thinking we should include a subset of the most popular rather than trying to include all, since it would inevitably get out of sync. Fortunately, a user can still use styles not in the option UI by simply specifying it in the settings.json.

Reply:
It is also worth noting that any style selected here will be used in the style= option in the ODS statement regardless of whether the code is being submitted in Viya 4, Viya 3.5, or V9. So, if the style is not defined, you’ll get a warning in the log and a fallback style will be used. For example, “WARNING: Style BLAH not found; HTMLEncore style will be used instead.”

Reply:
A problem for enum is that it will produce warnings if user wants to specify a style in the settings.json file.

![WARNING](https://github.com/sassoftware/vscode-sas-extension/assets/96556502/1bd4d4bd-118e-4147-bcfb-e637dc9e77de)

Reply:
Yes, I did notice that. It still works, but yes, the warning could be alarming. I initially looked for a combobox where user could type in a value if the desired value did not already exist in the combobox, but it doesn't look like vscode supports that kind of control. Another alternative would be to add another string option, in addition to the combobox, which would allow user to type in any desired style. If they typed a value in this "override HTML style" option, then it would override any value specified in the combobox. Wasn't crazy about needing two UI options for user to specify a style, but could run this by @skyeaussie and @crlewi.

Reply:
Added a number of other styles recommended by @cleester4, so here is the list we are going to go with:
Aqua, Daisy, Dove, HighContrast, HighContrastLarge, HTMLBlue, HTMLEncore, Ignite, Illuminate, Journal, Journal2, Journal3, Marine, Midnight, Moonflower, Opal, Pearl, Sapphire

The list above are all the styles defined on Viya 4 by default, except for these (which are ideally for other ODS destinations or otherwise don't fit HTML5 well):
Default, EGDefault, Excel, ExcelAqua, ExcelHighContrast, ExcelIgnite, ExcelIlluminate, ExcelMarine, ExcelMidnight, ExcelOpal, ExcelUmstead, Listing, PearlJ, Plateau, PowerPointAqua, PowerPointDark, PowerPointHighContrast, PowerPointIgnite, PowerPointIlluminate, PowerPointLight, PowerPointMarine, PowerPointMidnight, PowerPointOpal, PowerPointUmstead, Printer, Raven, Rtf

As noted above, any styles (built-in or user-defined) can still be used by explicitly specifying in the settings.json file. For example...
"SAS.results.html.style": "EGDefault"

For now, not going to worry about the warning that appears when you specify a value in the settings.json that does not correspond with an enum, since it is just a warning and still works.